pagopa/io-app
# fix: [IOBP-219] Initiative list into card details

## Short description
This PR fixes a regression introduced with the PR #4954 about the updated list for each payment method

## List of changes proposed in this pull request
- Updated reducer to refresh initiative list for a new payment method

## How to test
Open two or more payment method and check that the status of each initiative is different from each other.

## Acknowledgement
@forrest57
